xbmc/interfaces/json-rpc/PlayerOperations.cpp

```diff
@@ -569,7 +569,9 @@ JSONRPC_STATUS CPlayerOperations::Open(const std::string &method, ITransportLaye
       return InvalidParams;
 
     CFileItemList *l = new CFileItemList; //don't delete,
-    l->Add(std::make_shared<CFileItem>(channel));
+    CFileItem item(channel);
+    CServiceBroker::GetPVRManager().FillStreamFileItem(item);
```

Comment:
Please apply a similar fix for recordings: https://github.com/xbmc/xbmc/blob/ba1c25a90b5456bba8e7a7b826bdd31dbd024c65/xbmc/interfaces/json-rpc/PlayerOperations.cpp#L593